### ISSUE TITLE
Mocked Server is Not Using HTTPS

### ISSUE THREAD

**Issue by arctouch-matheusfaleiro** [andrzejchm/RESTMock#84]
The Mocked Server that is set up by RESTMock is using HTTP. Android SDK 28 will start to deny any connection between servers that are not using HTTPs. So, all the applications tests that are using SDK 28 will fail. Because they can not find the Mocked Response, the instrumented tests are going to crash.

**Comment by andrzejchm** [comment]
thanks for pointing this out! Will take care of it in the next release

**Comment by szymon-kazmierczak** [comment]
Thats not entirely true. You can enable HTTP connections using a debug security_config with the following:
```
<network-security-config>
    <domain-config cleartextTrafficPermitted="true" >
        <domain includeSubdomains="true">localhost</domain>
    </domain-config>
</network-security-config>
```

But I agree, having RESTMock communicate via HTTPS would be great - whether using self signed certs or otherwise!

**Comment by andrzejchm** [comment]
this should now be available in 0.3.2 release. Please reffer to[**"HTTPS"**](README.md#https) for more info
